src/roles/util.ts

```diff
@@ -123,20 +130,38 @@ export class Map {
   static roleFromWeaviate = (role: WeaviateRole): Role => PermissionsMapping.use(role).map();
 
   static roles = (roles: WeaviateRole[]): Record<string, Role> =>
-    roles.reduce((acc, role) => {
-      acc[role.name] = Map.roleFromWeaviate(role);
-      return acc;
-    }, {} as Record<string, Role>);
+    roles.reduce(
+      (acc, role) => ({
+        ...acc,
+        [role.name]: Map.roleFromWeaviate(role),
+      }),
+      {} as Record<string, Role>
+    );
 
   static users = (users: string[]): Record<string, User> =>
-    users.reduce((acc, user) => {
-      acc[user] = { id: user };
-      return acc;
-    }, {} as Record<string, User>);
+    users.reduce(
+      (acc, user) => ({
+        ...acc,
+        [user]: { id: user },
+      }),
```

Comment:
Nice syntactical change!

Reply:
Saw it on SO and thought it looks [idiomatic](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExN2h1b3I2Yng5YnRnN2t6cWJrdnFzNjZybnZoYWxmajByNTQxd2x2dyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3oriePYGXYh3bPg0Pm/giphy.gif) :)